lib/pbench/test/unit/server/query_apis/test_query_builder.py

```diff
@@ -0,0 +1,240 @@
+from pbench.server.api.resources import Schema
+import pytest
+from pbench.server.api.auth import Auth
```

Comment:
These should be reordered.

Reply:
I suspect I let vscode pull in `Schema` and forgot to go back and check the order. Ah, well. It seems to usually get them wrong; but being able to just `ctrl-.` instead of jumping to the top and typing out the import is *so* nice.